cg/constants/constants.py

```diff
@@ -53,3 +53,5 @@ class DataDelivery(StrEnum):
 
 STATUS_OPTIONS = ("affected", "unaffected", "unknown")
 ANALYSIS_TYPES = ["tumor_wgs", "tumor_normal_wgs", "tumor_panel", "tumor_normal_panel"]
+
+SARS_COV_REGEX = "^\d{2}CS\d{6}"
```

Comment:
`SARS_COV_REGEX = "^\d{2}CS\d{6}$"`

Also match end of line? https://blog.finxter.com/python-regex-start-of-line-and-end-of-line/

Reply:
done